packages/guide/src/components/ExternalLink.tsx

```diff
@@ -0,0 +1,12 @@
+import { FiExternalLink } from 'react-icons/fi';
+
+export function ExternalLink({ href, title }: { href: string; title: string }) {
+	return (
+		<a className="color-blurple flex content-center items-center justify-center  font-semibold" href={href}>
+			<div className="flex space-x-2">
+				<p>{title}</p>
+				<FiExternalLink size={22} />
+			</div>
+		</a>
```

Comment:
A `div` inside an anchor is semantically incorrect HTML, why not just apply the `space-x-2` on the anchor?

Additionally: `text-blurple` and theres a space too much after `justify-center`